net-im/signal-desktop-bin/signal-desktop-bin-7.43.0.ebuild

```
@@ -81,11 +83,10 @@ src_install() {
 	fperms u+s /opt/Signal/chrome-sandbox
 	pax-mark m opt/Signal/signal-desktop opt/Signal/chrome-sandbox opt/Signal/chrome_crashpad_handler
 
-	if use pulseaudio ; then
-		dosym -r /opt/Signal/${MY_PN} /usr/bin/${MY_PN}
-	else
-		dobin "${FILESDIR}/${MY_PN}"
-	fi
+	newbin - signal-desktop <<- _EOF_
+		#!/bin/sh
+		exec $(command -pv apulse) ${EPREFIX}/opt/Signal/signal-desktop "${@}"
```

The `$` need to be escaped, otherwise they will be evaluated during ebuild installation instead of runtime of the launcher.

```suggestion
		exec \$(command -pv apulse) ${EPREFIX}/opt/Signal/signal-desktop "\${@}"
```